lex/Makefile

```diff
@@ -54,16 +54,16 @@ clean::
 	rm -f $(programs) $(programs:=.exe)
 	rm -f *.cmo *.cmi *.cmx *.cmt *.cmti *.o *.obj
 
-parser.ml parser.mli: parser.mly
-	$(CAMLYACC) -v parser.mly
+%.ml %.mli: %.mly
+	$(OCAMLYACC) $(OCAMLYACCFLAGS) $<
```

Comment:
In lex/Makefiles: the rules for  `.mll` and `mly` are duplicated from `Makefile.common` and they can be removed.